lib/disposable.dart

```diff
@@ -12,5 +12,14 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-export 'package:w_common/src/disposable/disposable.dart';
-export 'package:w_common/src/disposable/disposable_manager.dart';
+/// This entry point has been split into separate VM and browser entry points;
+/// use disposable_vm.dart or disposable_browser.dart instead.
+@deprecated
+export 'package:w_common/src/disposable/disposable_vm.dart'
```

Comment:
Would it make sense to just `export : 'package:w_common/disposable_vm.dart'`? If that's possible then you wouldn't have to keep the two files in sync.